tests/api-compliance/localhost-config.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
This is nearly identical to `broker-config.json` (formerly named `config.json`). The only difference is the host that each of these address-- `broker` (compose) or `localhost` (circle).

The rationale for this is the same as explained in [this comment](https://github.com/Azure/open-service-broker-azure/pull/369/files#r177846967).